src/tests/unittests/validation/CopyCommandsValidationTests.cpp

```diff
@@ -314,6 +314,30 @@ TEST_F(CopyCommandTest_B2T, IncorrectRowPitch) {
     }
 }
 
+// Test B2T copies with incorrect buffer offset usage
+TEST_F(CopyCommandTest_B2T, IncorrectBufferOffset) {
+    uint32_t bufferSize = BufferSizeForTextureCopy(4, 4, 1);
+    nxt::Buffer source = CreateFrozenBuffer(bufferSize, nxt::BufferUsageBit::TransferSrc);
+    nxt::Texture destination = CreateFrozen2DTexture(16, 16, 5, nxt::TextureFormat::R8G8B8A8Unorm,
+                                                     nxt::TextureUsageBit::TransferDst);
+
+    {
```

Comment:
It is ok to have redundant control cases if it helps readability by having locally an example of what works.